.vscode/launch.json

```diff
@@ -23,6 +23,12 @@
             "program": "${workspaceRoot}/src/index.js",
             "cwd": "${workspaceRoot}",
             "preLaunchTask": "build"
+        },
```

Comment:
You may want to talk to @olydis regarding how to debug the typescript validator too